training/src/main/scala/com/airbnb/aerosolve/training/SplineTrainer.scala

```diff
@@ -352,7 +510,7 @@ object SplineTrainer {
                   dropout : Double,
                   margin : Double) : Double = {
     val flatFeatures = Util.flattenFeatureWithDropout(fv, dropout)
-    val prediction = model.scoreFlatFeatures(flatFeatures)
+    val prediction = model.scoreFlatFeatures(flatFeatures) / (1.0 - dropout)
```

Comment:
Ok let me revert this tomorrow then since there are additional calibration steps and it doesn't change the performance AUC wise before.